contrib/k8s/README.md

```diff
@@ -27,17 +37,23 @@ dex needs a secret key for encrypting private keys in the database. These can be
 
 [k8s-secrets]: http://kubernetes.io/v1.0/docs/user-guide/secrets.html
 
-```
-kubectl create -f dex-secrets.yaml
+The secret will be created as part of the next command, from `dex-overlord.yaml`
+
+If you would like to set up your own secret, you can do the following:
+
+```bash
+# For a real secret (i.e. not checked into this public repo), run the following
+# and comment out the secret in the dex-overlord.yaml file.
+secret=$(dd if=/dev/random bs=1 count=32 2>/dev/null | base64 | tr -d '\n' > key-secrets)
```

Comment:
The internal command doesn't print anything. You need to remove `> key-secrets` so it prints the secret.